neo/Network/RPC/RpcServer.cs

```diff
@@ -314,6 +314,20 @@ private JObject Process(string method, JArray _params)
                         json["unverified"] = new JArray(unverifiedTransactions.Select(p => (JObject) p.Hash.ToString()));
                         return json;
                     }
+                case "getmempoolstatisticsnextblock":
+                    {
+                        List<Fixed8>  mempoolStatistics = Blockchain.Singleton.MemPool.GetStatisticsForNextBlock();
+                        JObject json = new JObject();
+                        json["sortedHP_avgNetFees"] = new JArray(mempoolStatistics[0].ToString());
+                        json["sortedLP_avgNetFees"] = new JArray(mempoolStatistics[1].ToString());
+                        json["unverifiedHP_avgNetFees"] = new JArray(mempoolStatistics[2].ToString());
```

Comment:
What is the value of having the verified and unverified transactions reported separately? 

Can we just merge them both for the purpose of reporting statistics and not have the distinction of which type they are. Some nodes may have already reverified and some may have not. If they are in the pool at all they were valid at some point, and there is a good chance they are still valid.

If we aren’t going to precalculate statistics in memory, we should allow passing a count of top items to perform the statistics for min, max, median, average for the top `count` items from combined verified and unverified transactions. Median may be more meaningful than average as high fee outliers could skew mean.

Reply:
Sounds good, Jeff.
Makes sense to merge both.

About other metrics, I agree in implementing other ones.